Doc/cmus.txt

```diff
@@ -1532,6 +1539,9 @@ CMUS_CHARSET
 CMUS_HOME
 	Override cmus config directory (default: $XDG_CONFIG_HOME/cmus).
 
+CMUS_PLAYLIST_DIR
+	Override the cmus playlist directory (default: $XDG_CONFIG_HOME/cmus/playlists).
+
```

Comment:
If taken stricktly this is also not related to the pl-flush/reload functionality. One issue per PR.

Reply:
Removed.